lib/src/main/java/org/team100/lib/motor/SimulatedMotor.java

```diff
@@ -41,17 +42,16 @@ public void setDutyCycle(double output) {
     public void stop() {
         m_velocity = 0;
     }
-
     /**
-     * Ignores accel, because the simulated motor responds instantly to the velocity
-     * command, i.e. the accel is effectively infinite.
+     * Does not use feedforward as I am not sure how that would work with a fake motor at the moment
      */
     @Override
     public void setVelocity(double velocity, double accel) {
         if (Double.isNaN(velocity))
             throw new IllegalArgumentException("velocity is NaN");
-        m_velocity = velocity;
-        // ignore accel
+        double output = controller.calculate(m_velocity,velocity);
```

Comment:
could you say something about what problem this solves, and how you know if it's solved?